src/components/Section/Citizenship/Intro/index.js

```diff
@@ -0,0 +1,2 @@
+import Intro from './Intro'
+export default Intro
```

Comment:
Do we want to keep up this pattern of exporting from a component directory's `index.js`? I'm happy either way, just want to know if I should be doing this also!

Reply:
I'd prefer:
```
Intro
    index.jsx
    index.test.jsx
```

I just followed the same format for the sake of consistency.